.github/workflows/selfdrive_tests.yaml

```diff
@@ -274,7 +274,7 @@ jobs:
       timeout-minutes: 40
       run: |
         ${{ env.RUN }} "export SKIP_LONG_TESTS=1 && \
-                        $PYTEST -n auto --dist=loadscope --timeout 30 -o cpp_files=test_* && \
+                        $PYTEST -n auto --dist=loadscope --setup-timeout 15 --execution-timeout 30 --teardown-timeout 15 -o cpp_files=test_* && \
```

Comment:
why isn't this in the default config?

Reply:
it was because we didn't have pytest-timeout in agnos yet. But we do have pytest-timeouts so I moved it